src/molecules/modal/Modal.html

```diff
@@ -1,19 +1,20 @@
 <transition :name="modalMainTransition">
-  <div
+  <component
+    :is="modalTag"
     v-show="visible"
     :aria-hidden="ariaHidden"
     :aria-labelledby="ariaLabelledby"
     :aria-describedby="ariaDescribedby"
     class="a-modal"
-    role="dialog"
+    :role="modalRole"
   >
     <div
       @click.self="handleBackgroundClick"
       class="a-modal__mask"
     ></div>
     <transition :name="modalContentTransition">
       <div
-        class="a-modal__content"
+        :class="['a-modal__content', modalContentClass]"
```

Comment:
do we need `modalContentClass` prop?

Reply:
Imo it's beneficial but for future usage, now it's not really needed - not used anywhere